.github/workflows/update-gpuci.yml

```diff
@@ -0,0 +1,51 @@
+name: Check for gpuCI updates
+
+on:
+  schedule:
+    - cron: "0 * * * *"
+  workflow_dispatch:
+
+jobs:
+  update-gpuci:
+    runs-on: ubuntu-latest
+    if: github.repository == 'dask/dask'
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Get latest cuDF nightly version
+        id: latest_version
+        uses: jacobtomlinson/gha-anaconda-package-version@0.1.3
+        with:
+          org: "rapidsai-nightly"
+          package: "cudf"
+          version_system: "CalVer"
+
+      - name: Strip git tags from versions
+        run: echo "RAPIDS_VER=${FULL_RAPIDS_VER::-10}" >> $GITHUB_ENV
+        env:
+          FULL_RAPIDS_VER: ${{ steps.latest_version.outputs.version }}
```

Comment:
Nitpick: you might consider moving `run:` to be after `env:` to make it more clear to readers where `FULL_RAPIDS_VER` is coming from

Reply:
Sure! That makes sense